props/saber_BC_buttons.h

```diff
@@ -638,6 +699,53 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
     }
   }
 
+#ifdef DUAL_BLADES
+  void ControlBladeState(bool& this_blade_on_, bool other_blade_on_, bool& this_blade_first_, bool& other_blade_first_, uint16_t THIS_BLADE, uint16_t OTHER_BLADE, bool is_main_blade_) {
```

Comment:
Didn't you make a class for grouping the bools that belong to one blade together?